src/wallet/KintoWalletFactory.sol

```diff
@@ -312,7 +320,10 @@ contract KintoWalletFactory is Initializable, UUPSUpgradeable, OwnableUpgradeabl
 }
 
 contract KintoWalletFactoryV22 is KintoWalletFactory {
```

Comment:
shouldnt this be v23?

Reply:
I usually bump version in the deploy PR, but fair enough.